libraries/AP_HAL_ChibiOS/hwdef/Atlas-Control/hwdef.dat

```diff
@@ -0,0 +1,307 @@
+# hw definition file for processing by chibios_hwdef.py for Atlas-Control
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_ATLAS_CONTROL
+
+FLASH_SIZE_KB 2048
+
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART2 USART6 USART1 UART4 UART8 UART7 OTG2
```

Comment:
since you dont pin out UART8, you should use EMPTY instead and not define its pins below